community/kernel/src/main/java/org/neo4j/kernel/impl/store/NeoStores.java

```diff
@@ -67,7 +68,7 @@ public static boolean isStorePresent( PageCache pageCache, File storeDir )
         }
     }
 
-    private enum StoreType
+    public enum StoreType
```

Comment:
i tried to do that as well, but instead of already not that minimal change it becomes much bigger, so would propose to do that in 3.0 on top of other work as soon as it will be merged